packages/boolean/StoryComponent.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { Fragment } from 'react'
+import useBoolean from '.'
+
+export default () => {
+  const [value, toggle] = useBoolean(true)
+
+  return (
+    <Fragment>
+      <p>The value is now {value ? 'true' : 'false'}</p>
+      <button onClick={() => toggle()}>Toggle</button>
```

Comment:
```suggestion
      <button onClick={toggle}>Toggle</button>
```

Reply:
I had it like this, but TS complains that the type `event` is incompatible with `nextValue`. Thinking about it, this would effectively pass a truthy value every time, and it wouldn't work as intended

Reply:
Yep your right, my bad! 😃